DataPlotly/core/plot_types/box.py

```diff
@@ -43,7 +43,7 @@ def create_trace(settings):
             y = settings.y
 
         return [graph_objs.Box(
-            x=x,
+            x=x if x else None,
```

Comment:
One shorthand python trick is:

    x = x or None

Because if the first value (x) is false, then the second condition gets evaluated. And the statement returns the actual value of each operand.